packages/template-retail-react-app/app/components/experience/image-tile/index.jsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Image} from '@chakra-ui/react'
+
+/**
+ * Simple Image tile component that can be used inside any Layout component
+ * @param image Object containing the image url, _type and focalPoint.
+ * @param rest The rest of the potential image parameters.
+ * @returns {JSX.Element}
+ */
+const ImageTile = ({image, ...rest}) => {
+    return (
+        <figure className={'image-tile-figure'}>
+            <picture>
+                <source srcSet={image?.src?.tablet} media="(min-width: 48em)" />
+                <source srcSet={image?.src?.desktop} media="(min-width: 64em)" />
+                <Image
+                    className={'image-tile-image'}
+                    data-testid={'image-tile-image'}
+                    src={image?.src?.mobile ? image?.src?.mobile : image?.url}
+                    ignoreFallback={true}
+                    alt={image?.alt}
+                    title={image?.alt}
+                    {...rest}
```

Comment:
Do we need this "rest" object? For the most part this component isn't going to be used directly by a developer, it'll only be created by the Page/Region components, so we don't need this ability to passthrough props. Especially when it makes defining your PropTypes impossible.